src/components/screen/desktop.js

```diff
@@ -502,6 +531,11 @@ export class Desktop extends Component {
                     {this.renderWindows()}
                 </div>
 
+                {/* Window Area */}
+                <div className="absolute h-full w-full bg-transparent" data-context="desktop-area">
+                    {this.renderCalcWindows()}
+                </div>
+
```

Comment:
remove this also,
we have already implemented that, so no need to re-render all apps again.